doc/user/content/sql/create-source/kafka.md

```diff
@@ -187,14 +187,14 @@ SELECT key,
 FROM (SELECT key,
              field1,
              field2,
-             unnest(headers).key AS thekey,
-             unnest(headers).value AS value
+             unnest(headers).key AS header_key,
+             convert_from(unnest(headers).value, 'utf8') AS header_value
       FROM mv_kafka_metadata) AS km
-WHERE thekey = 'kvalue';
+WHERE header_key = 'trace_id';
```

Comment:
This seems to be working. Not sure if it can be further simplified.

```
SELECT
...
(SELECT convert_from(value, 'utf8') FROM (SELECT unnest(headers).key as key, unnest(headers).value as value) WHERE key='trace_id')
FROM ...;
```

Reply:
Hah. This might work!

```sql
SELECT
...
(
    SELECT convert_from((h).value, 'utf8') AS trace_id
    FROM unnest(headers) h 
    WHERE (h).key = 'trace_id'
)
FROM ...;
```

Reply:
Yes, sorry! That's exactly the right way to do it. I think you can also write it like:

```sql
SELECT
...
(
    SELECT convert_from(value, 'utf8') AS trace_id
    FROM unnest(headers) h (key, value) 
    WHERE h.key = 'trace_id'
)
FROM ...;
```